packages/openneuro-components/src/activity-slider/ActivitySlider.tsx

```diff
@@ -1,96 +1,139 @@
 import formatDistanceToNow from 'date-fns/formatDistanceToNow'
 import parseISO from 'date-fns/parseISO'
-import Slider from 'react-slick'
-
-// Import css files
-import 'slick-carousel/slick/slick.css'
-import 'slick-carousel/slick/slick-theme.css'
+import { Link } from 'react-router-dom'
+import Carousel from 'react-multi-carousel'
+import 'react-multi-carousel/lib/styles.css'
 
 import './slider.scss'
 
 export interface ActivitySliderProps {
-  sliderClass?: string
-  dots?: boolean
-  responsive?: object
-  slidesToShow?: number
-  slidesToScroll?: number
-  swipeToSlide?: boolean
-  slideHeader?: string
-  data: Record<string, any>
+  className?: string
+  showDots?: boolean
+  infinite?: boolean
+  keyBoardControl?: boolean
+  containerClass?: string
+  itemClass?: string
+
+  responsive: {
+    superLargeDesktop?: {
+      // the naming can be any, depends on you.
+      breakpoint?: { max?: number; min?: number }
+      items?: number
+    }
+    desktop?: {
+      breakpoint?: { max?: number; min?: number }
+      items?: number
+    }
+    tablet?: {
+      breakpoint?: { max?: number; min?: number }
+      items?: number
+    }
+    mobile?: {
+      breakpoint?: { max?: number; min?: number }
+      items?: number
+    }
+  }
+  data: {
+    id: string
+    analytics: {
+      views: number
+    }
+    latestSnapshot: {
+      tag: string
+      description: {
+        Name: string
+      }
+      summary: {
+        modalities: string[]
+      }
+    }
+  }[]
 }
+const LeftArrow: React.FC = ({ onClick }) => (
+  <i className="fas fa-chevron-left" onClick={() => onClick()} />
+)
+
+const RightArrow: React.FC = ({ onClick }) => (
+  <i className="fas fa-chevron-right" onClick={() => onClick()} />
+)
 
 export const ActivitySlider: React.FC<ActivitySliderProps> = ({
-  slideHeader,
-  sliderClass,
+  className,
   data,
-  accessibility = true,
-  draggable = true,
-  ...props
+  showDots,
+  slideHeader,
+  responsive,
+  infinite,
+  keyBoardControl,
+  containerClass,
+  itemClass,
 }) => {
-  const datasets = data
   return (
-    <div className={'container activity-slider' + ' ' + sliderClass}>
+    <div className={'activity-slider' + ' ' + className}>
       <h3>{slideHeader}</h3>
-      <Slider {...props}>
-        {datasets.map((item, index) => (
-          <>
-            <a
-              href={
-                'datasets/' +
-                item.node.id +
-                '/versions/' +
-                item.node.latestSnapshot.tag
-              }>
-              <div className="activity-slider-node" key={index}>
-                <div className="ds-name">
-                  <h4>{item.node.latestSnapshot.description.Name}</h4>
-                </div>
+      {/* @ts-expect-error */}
+      <Carousel
+        infinite={infinite}
+        keyBoardControl={keyBoardControl}
+        containerClass={containerClass}
+        itemClass={itemClass}
+        showDots={showDots}
+        responsive={responsive}
```

Comment:
Do you know how this library handles responsive? I think we'll want this to render in SSR mode and my guess is it won't (can't test due to broken SSR on master).

Reply:
The front page w/out cookies set is getting some styling from openneuro-components it looks like the css switch for enabled/disabled is no longer working?

the multi carousel module does have support for ssr
https://www.npmjs.com/package/react-multi-carousel